src/juxt.ts

```diff
@@ -0,0 +1,58 @@
+import type Append from "./types/Append";
+import type Arrow from "./types/Arrow";
+import type Cast from "./types/Cast";
+import type Tail from "./types/Tail";
+
+type HasItem<T extends any[]> = T extends [any, ...any] ? true : false;
```

Comment:
It may be good to create a separate file

Reply:
The `HasItem` type has been separated into a separate file.